hal/halx86/amd64/x86bios.c

```diff
@@ -291,7 +326,9 @@ ValidatePort(
         case 0x3B6: return (Size <= 2);
     }
 
-    return FALSE;
+    /* Allow but report unknown ports, we trust the BIOS for now */
```

Comment:
What's happening in case you continue to return FALSE? Does the boot fail in some cases?

Reply:
Yes, on some Hw